### PR TITLE
Entries: Use "properties" when referring to individual options

### LINKED ISSUE #247: Use consistent noun to refer to "nested options" (properties of an option)
We currently use both "settings" and "properties" when talking about options. Since we also use "setting" as a verb, I prefer to avoid that when referring to properties of an option, like properties of the `classes` option.

Should be a simple search and replace (or not, as usual). Would like some input before making the change though. /cc @agcolom @arthurvr @kswedberg

### PR DESCRIPTION
**Pull request by jzaefferer**
Removes some ambiguity since we also "setting" as a verb.

Fixes #247

### CODE REVIEW

File: entries/accordion.xml [hunk 1]
```diff
@@ -78,7 +78,7 @@
 				<desc>Name of <a href="/easings/">easing</a> to use with default duration.</desc>
 			</type>
 			<type name="Object">
-				<desc>Animation settings with <code>easing</code> and <code>duration</code> properties.
+				<desc><code>easing</code> and <code>duration</code> properties to configure animations.
```
Comment (scottgonzalez) [inline entries/accordion.xml]: Perhaps "An object containing `easing`..."

**Comment by scottgonzalez** [comment]
One minor suggestion, but other than that, this looks good.

**Comment by jzaefferer** [comment]
Updated

**Comment by scottgonzalez** [comment]
Looks good.

**Comment by jzaefferer** [comment]
Also checked the 1-12 branch, no regressions there.

**Pull request closed without merging**